src/wiki/views/article.py

```diff
@@ -574,11 +573,11 @@ class Source(ArticleMixin, TemplateView):
 
     @method_decorator(get_article(can_read=True))
     def dispatch(self, request, article, *args, **kwargs):
-        return super(Source, self).dispatch(request, article, *args, **kwargs)
+        return super().dispatch(request, article, *args, **kwargs)
 
     def get_context_data(self, **kwargs):
         kwargs['selected_tab'] = 'source'
-        return ArticleMixin.get_context_data(self, **kwargs)
+        return super().get_context_data(**kwargs)
```

Comment:
This one is calling `ArticleMixin.get_context_data` explicitly to avoid calling the `get_context_data` of the wrong parent class.

Reply:
Yes, I'll revert that change.